.github/workflows/contracts.yml

```diff
@@ -33,22 +36,23 @@ jobs:
       - name: Prepare target
         run: rustup target add wasm32-unknown-unknown
 
-      # This will attempt to build all contracts in the workspace.
-      # This command will likely change when we have more than contracts in the workspace
-      - name: Build All
-        run: cargo build --release --target wasm32-unknown-unknown --verbose
+      - name: Build
+        run: cargo build -p ${{ matrix.contract }} --release --target wasm32-unknown-unknown --verbose
 
       - name: Lint
-        run: cargo clippy --all-targets --all-features
+        run: cargo clippy -p ${{ matrix.contract }} --all-targets --all-features
       - name: Upload compiled contracts
         uses: actions/upload-artifact@v3
         with:
           name: wasm-files
```

Comment:
Is this summing up the contracts or replacing the entire `wasm-files` artifact each time ?

Reply:
These are artifacts attached to each build. and not related to OCI

Reply:
Yes, my question goes more in the line what happens, when the matrix is executed. Are this artifacts being replaced ? Whats the content of the `wasm-files` artifact at the end of this job ? Is a single contract or all the contracts ?